src/components/ContextBar.js

```diff
@@ -101,6 +113,17 @@ export const ContextBar = ({
     ])
   }
 
+  //This excludes cromwellapp in the calculation.
+  const getTotalToolAndDiskCostDisplay = () => {
+    const galaxyApp = getCurrentApp(tools.Galaxy.appType)(apps)
+    const galaxyDisk = getCurrentAppDataDisk(tools.Galaxy.appType, apps, appDataDisks, workspaceName)
+    const galaxyCost = galaxyApp && galaxyDisk ? getGalaxyCost(galaxyApp, galaxyDisk) : 0
+    const runtimeCost = currentRuntime ? getRuntimeCost(currentRuntime) : 0
+    const curPd = getCurrentPersistentDisk(runtimes, persistentDisks)
+    const diskCost = curPd ? getPersistentDiskCostHourly(curPd, computeRegion) : 0
+    return `${Utils.formatUSD(galaxyCost + runtimeCost + diskCost)}`
```

Comment:
Why is this in US dollars? If someone is using Montreal, will the correct costs be displayed?

Reply:
For clarity, the IA team will follow up about localization/currency in a future ticket.